src/components/QuestionView/Expanded.tsx

```diff
@@ -31,8 +31,8 @@ export default function ExpandedView({
       : {};
 
   return (
-    <Card shadow="sm" p="lg" my="sm" withBorder className="secondaryCard slideDown">
-      <Card className="secondaryCard clickable" p={0} onClick={() => setExpandedQuestionIndex(null)}>
```

Comment:
Same question. We should be able collapse by clicking on the header like today.

Reply:
fixed